.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+
+<!-- Please fill out any relevant sections and remove those that don't apply -->
+## Motivation
+<!-- Why was this pull request needed? -->
+## Description of changes
+<!-- What was changed in this pull request? -->
+## Breaking changes
```

Comment:
Should we have a "How to test/qa version"?

Reply:
As discussed in vc, we won't add this to the template as we don't want to encourage manual testing, it can be added as needed.